src/node_quic_session.cc

```diff
@@ -1558,6 +1559,21 @@ bool QuicSession::ReceivePacket(
       case NGTCP2_ERR_RECV_VERSION_NEGOTIATION:
         break;
       default:
+        // Per ngtcp2: If NGTCP2_ERR_RETRY is returned,
+        // QuicSession must be a server and must perform
+        // address validation by sending a Retry packet
+        // then immediately close the connection.
+        if (err == NGTCP2_ERR_RETRY && IsServer()) {
+          // TODO(@jasnell): Need to verify proper ordering of
+          // scid and rcid here
+          Socket()->SendRetry(
+              GetNegotiatedVersion(),
+              &QuicCID(scid()),
+              &QuicCID(rcid()),
```

Comment:
`&`-ing a local variable is usually a sign that the function should be taking a const or rvalue reference, i.e. `const QuicCID&` rather than `QuicCID*` here … does that make sense here?

Reply:
Yep, absolutely. Making the change throughout...